tests/builder/help.rs

```diff
@@ -246,6 +246,41 @@ OPTIONS:
 
 static HIDE_POS_VALS: &str = "ctest 0.1
 
+USAGE:
+    ctest [OPTIONS]
+
+OPTIONS:
+    -c, --cafe <FILE>    A coffeehouse, coffee shop, or café.
+    -h, --help           Print help information
+    -p, --pos <VAL>      Some vals [possible values: fast, slow]
+    -V, --version        Print version information
+";
+#[cfg(feature = "unstable-v4")]
+static POS_VALS_HELP: &str = "ctest 0.1
+
+USAGE:
+    ctest [OPTIONS]
+
+OPTIONS:
+    -c, --cafe <FILE>
+            A coffeehouse, coffee shop, or café.
+
+    -h, --help
+            Print help information
+
+    -p, --pos <VAL>
+            Some vals
+
+            Possible values:
+                - fast
+                - slow: not as fast
+
+    -V, --version
+            Print version information
+";
+#[cfg(not(feature = "unstable-v4"))]
+static POS_VALS_HELP: &str = "ctest 0.1
```

Comment:
nit: can you move these into the test body?  I've been writing new tests like that because I find it easier to read and update but haven't had time to update all of the old tests

Reply:
makes sense, I did that for the wrap help test, as the other wrap ones did, but I can do that here as well.